### ISSUE TITLE
The installer media app doesn't work correctly.

### ISSUE THREAD

**Issue by Mtkhan13** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#182]
Hi friend, I have tried both version of this creator. V2 and V3 but "Install Media HP Laptop OC" app stuck right after usb remaining to "Install macOS Monterey". I'm not sure what causing this. I have tried on my other system with Ventura installed same issue there as well. I checked in the activity monitor and its appears as not responding. 

![Screenshot 2023-08-07 at 2 39 28 AM](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/22547975/ab5f6dfb-9171-417e-9b1b-eed0c9b5f0c7)

**Comment by Mtkhan13** [comment]
Also tried the clover variant it also shows the same issue. Not sure what causing it please guide. Thanks
![Screenshot 2023-08-07 at 2 52 24 AM](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/22547975/8688af5b-1233-430f-abe7-1e27c17937d8)

**Comment by chris1111** [comment]
The app not stop you have to wait Then its normal  Not Responding

Normal USB can take 25 30 minutes

**Comment by Mtkhan13** [comment]
Ah I see let me try again

**Comment by chris1111** [comment]
> Ah I see let me try again

Normal USB can take 25 30 minutes then more depend the speed of USB 2 or USB 3
